wiki/BanchoBot/id.md

```diff
@@ -1,29 +1,27 @@
 
-## Tampilan dan Tugas
+<!-- BanchoBot (server) commands start with an exclamation mark! `!`-->
+## Perintah BanchoBot
 
-### !help
+Untuk menggunakan perintah BanchoBot, anda harus menggunakan awalan `!` setiap saat. **Pengguna normal hanya bisa menggunakan perintah ini di kanal multiplayer dan di PM BanchoBot.** Jika pengguna mengirim perintah dalam obrolan publik, pengguna biasa tidak akan melihatnya dan mereka mendapatkannya sebagai pesan pribadi. Pengguna juga dapat menggunakan perintah klien `/bb <command>` untuk membuka tab BanchoBot secara otomatis dan mengirim perintah dengan segera.
 
--   Membuka tab Banchobot di konsol chatmu. Ia akan memberikanmu daftar perintah seperti yang tertulis di tabel.
-    -   Jika kamu membuka tab Banchobot, apapun perintah yang kamu gunakan akan tampil di tab Banchobot di tab manapun kamu berada ketika kamu menggunakan perintah. (kecuali operator kanal)
-    -   Jika kamu mengirimkan perintah tanpa tanda seru ("!") jika sedang berada di tab Banchobot, BanchoBot masih menerima perintah yang sama ketika kamu menggunakan /bb
--   Kamu tidak bisa menggunakan perintah yang menggunakan tanda seru di tab selain Pesan Pribadi Banchobot.
+*Untuk daftar perintah yang menggunakan slash (klien), kunjungi artikel [Konsol Obrolan](/wiki/Chat_Console#commands-list).*
 
-#### Perintah Standar (!PERINTAH or /bb PERINTAH)
-
-| PERINTAH | Efek | Contoh | Respon Banchobot |
-| -------- | ---- | ------ | ---------------- |
-| WHERE (pengunna) | Menampilkan lokasi pengguna yang tertulis | !where John | John is in USA |
-| STATS (pengunna) | Menampilkan status / keadaan pengguna yang tertulis (berdasarkan di tab pengguna berada) | !stats John | Stats for John is Idle. Score:00 (#0). Plays:2 (lvl 4). Accuracy:0.00%. |
-| FAQ (daftar kode)(item) (daftar) | Berbagai kegunaan. [Daftar] Daftar Perintah | !faq wiki / !faq ru:wiki | The osu! Wiki - Make it awesome! / Примите участие в заполнении вики! |
-| REPORT (alasan) | Memanggil sebuah moderator | !report Thomas offending comments | Chat moderators has been alerted. Thanks for your help. |
-| REQUEST (list) | Menampilkan sebuah permintaan mod secara acak. [list] Menampilkan 5 permintaan mod terbaru secara acak. | !request | Seether - Fake It by [Dellirium] |
-| ROLL (nomor) | Menggulingkan sebuah dadu (virtual/maya) dan mendapatkan hasil yang acak dari 1 hingga (nomor)(biasanya 100) | !roll 9000 | John rolls 1337 point(s) |
-
-*\*Alasan dalam perintah !REPORT harus menggunakan bahasa Inggris sebab Moderator Obrolan umumnya menggunakan bahasa Inggris.*
+<!-- Daftar Perintah -->
+| Perintah   | Penjelas            | Deskripsi | Contoh Masukan | Contoh Tanggapan |
+| --------- | ---------------------- | ----------- | ------------- | ---------------- |
+| `HELP`    | Tidak ada                   | Menampilkan daftar perintah yang tersedia                                                    | `!help`                               | Daftar ini |
```

Comment:
I'm not really sure whether the examples here needs to be translated as well or not, because (to my knowledge) as things stand out if you use a Bancho command in-game the response given will always be written in English regardless of what language of you're using in-game (i.e. typing "!where Kyubey" while using the Indonesian localization will always result in "Kyubey is in Russia" instead of "Kyubey berada di Rusia")